components/clientComponents/forms/Form/Form.tsx

```diff
@@ -405,15 +416,22 @@ export const Form = withFormik<FormProps, Responses>({
       if (result.error) {
         if (result.error.message.includes("FileValidationResult")) {
           formikBag.setStatus("FileError");
+        } else if (result.error.message.includes("Failed to find Server Action")) {
+          throw new Error("Failed to find Server Action");
```

Comment:
@dsamojlenko we can likely just re-throw vs creating a "new" error here

Thoughts?